### PR TITLE
Release 18.0.0

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
We should not release yet. There may still be more changes coming and especially with a major you want to ensure that no other breaking change goes in.

**Comment by wbclark** [comment]
> We should not release yet. There may still be more changes coming and especially with a major you want to ensure that no other breaking change goes in.

ACK. I converted this to a draft and we can update it when ready.

**Comment by wbclark** [comment]
Looks like there are still a few more PRs which could get merged before updating this and marking it ready for review

In particular it looks good to get https://github.com/theforeman/puppet-foreman_proxy/pull/652 and https://github.com/theforeman/puppet-foreman_proxy/pull/661 merged first if everything is ready for those

**Comment by wbclark** [comment]
Updated with latest commits.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+- Refs [\#31893](https://projects.theforeman.org/issues/31893) - make ansible callback configurable [\#662](https://github.com/theforeman/puppet-foreman_proxy/pull/662) ([evgeni](https://github.com/evgeni))
+- Mark compatible with Foreman 17.x [\#658](https://github.com/theforeman/puppet-foreman_proxy/pull/658) ([ekohl](https://github.com/ekohl))
+- Remove Puppet version check [\#657](https://github.com/theforeman/puppet-foreman_proxy/pull/657) ([ekohl](https://github.com/ekohl))
+- Add smart\_proxy\_dns\_route53 plugin support [\#656](https://github.com/theforeman/puppet-foreman_proxy/pull/656) ([Nevermore24](https://github.com/Nevermore24))
+- Add shellhooks plugin [\#651](https://github.com/theforeman/puppet-foreman_proxy/pull/651) ([adamruzicka](https://github.com/adamruzicka))
+
+**Fixed bugs:**
+
+- Refs [\#31893](https://projects.theforeman.org/issues/31893) - correct default value [\#663](https://github.com/theforeman/puppet-foreman_proxy/pull/663) ([evgeni](https://github.com/evgeni))
```
Comment (evgeni) [inline CHANGELOG.md:23]: Technically this can be skipped as the bug was in no release?

Comment (wbclark) [reply CHANGELOG.md:23]: Good point!

I have to step away urgently but will fix that up when I return

**Comment by ehelms** [comment]
I merged #652 -- feel free to update and then we can go with this. Since this depends on puppet-foreman_proxy which will get a major version bump I'll wait to also include any metadata updates needed.

**Comment by wbclark** [comment]
Updated with latest commits and requested changelogs

**Review by ehelms — APPROVED** [review]

**Comment by ekohl** [comment]
I should have made a note here, but I was aware https://github.com/theforeman/puppet-foreman_proxy/pull/664 was still going to come. That should have made it before it was released.

**Pull request merged**